src/phrases/phrases-nb.ts

```diff
@@ -33,6 +33,10 @@ const phrases = {
     create: 'Opprett nytt',
     isSelectedItem: 'Lagt til',
   },
+  accordion: {
+    openAll: 'Åpne alle',
+    closeAll: 'Lukk alle',
+  },
```

Comment:
Finnes ikkje disse allerede i ndla/ui? Treng vel ikkje definere dei på nytt då.

Reply:
tydligvis ikke. Måtte definere dem for at det skulle funke

Reply:
Dei finnes i ndla/ui men du har ikkje oppdatert den pakka. Derfor måtte du definere dei her. Om du bumper pakka treng du ikkje ha desse.

Reply:
se der ja